shared/img/toolbar-rating-a.svg

```diff
@@ -0,0 +1 @@
+<svg height="24" viewBox="0 0 24 24" width="24" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><circle cx="12" cy="12" fill="#3fa047" r="12"/><path d="m17.19 16.8h-2.415l-.63-1.695h-4.29l-.63 1.695h-2.415l3.855-10.005h2.67zm-3.63-3.57-1.56-4.305-1.56 4.305z" fill="#fff"/></g></svg>
```

Comment:
I realize it's not as important in the extension (where everything gets downloaded in one big bundle) as on the website, but it would be nice to make sure those SVGs are optimized. I usually use https://jakearchibald.github.io/svgomg/.

<img width="297" alt="smaller" src="https://user-images.githubusercontent.com/985504/54451736-c7b69480-4753-11e9-9c31-cfcda6822095.png">